octoprint_camera/__init__.py

```diff
@@ -651,9 +654,11 @@ def save_debug_img(img, name):
         do_lens = pic_type in (PIC_LENS, PIC_BOTH)
 
         if which == LAST:
-            img_jpg = self.camera_thread.get_latest_img()
+            img_jpg = (
+                self.camera_thread.get_latest_img() if self.camera_thread else None
+            )
         elif which == NEXT:
-            img_jpg = self.camera_thread.get_next_img()
+            img_jpg = self.camera_thread.get_next_img() if self.camera_thread else None
         else:
             raise Exception("We shouldn't be here, huhoo..")
```

Comment:
I know you didn't add this, but could you please change it? :sweat_smile:

Reply:
Yopu mean the exception??